diff -pruN bison-3.0.3.orig/doc/local.mk bison-3.0.3/doc/local.mk
--- bison-3.0.3.orig/doc/local.mk	2015-01-13 16:41:47.000000000 +0300
+++ bison-3.0.3/doc/local.mk	2015-01-22 13:01:38.074902930 +0300
@@ -112,18 +112,6 @@ if ! CROSS_COMPILING
 MAN_DEPS = doc/bison.help doc/bison.x $(top_srcdir)/configure
 endif
 
-$(top_srcdir)/doc/bison.1: $(MAN_DEPS)
-	$(AM_V_GEN)$(HELP2MAN)			\
-	    --include=$(top_srcdir)/doc/bison.x	\
-	    --output=$@.t src/bison$(EXEEXT)
-	$(AM_V_at)if $(remove_time_stamp) $@ >$@a.t 2>/dev/null &&	 \
-	   $(remove_time_stamp) $@.t | cmp $@a.t - >/dev/null 2>&1; then \
-	  touch $@;							 \
-	else								 \
-	  mv $@.t $@;							 \
-	fi
-	$(AM_V_at)rm -f $@*.t
-
 if ENABLE_YACC
 nodist_man_MANS = doc/yacc.1
 endif
