From c3f4942400668495e094679c6c9ea339c4a7fbd0 Mon Sep 17 00:00:00 2001
From: Dmitry Shifrin <dmitry.shifrin@cogentembedded.com>
Date: Wed, 28 Oct 2015 16:22:29 +0300
Subject: [PATCH] enable rmobile usb phy on silk

---
 include/configs/silk.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/silk.h b/include/configs/silk.h
index 120962a..a3212e7 100644
--- a/include/configs/silk.h
+++ b/include/configs/silk.h
@@ -193,6 +193,7 @@
 #define CONFIG_USB_EHCI
 #define CONFIG_USB_EHCI_R8A779x
 #define CONFIG_USB_MAX_CONTROLLER_COUNT	2
+#define CONFIG_USB_PHY_RMOBILE
 
 /* MMCIF */
 #define CONFIG_MMC			1
-- 
1.9.1

